### ISSUE TITLE
[devcontainers] Extension does not accept .devcontainer.json as a symlink

### ISSUE THREAD

**Issue by svenengstrom-vaderstad** [microsoft/vscode-remote-release#10536]
I have a few common configuration files stored in a submodule in my repo, among them is a devcontainer config file.
If i link to this file (`ln -sr <sub-module>/devcontainer.json <repo-root>/.devcontainer.json`) and open VS Code I don't get the "Folder contains a Dev Container configuration file"-popup. It does however work to create the symlink inside a .devcontainer-folder (`ln -sr <sub-module>/devcontainer.json <repo-root>/.devcontainer/devcontainer.json`).

To sum up:
 ✅ File named `.devcontainer.json`: Works
 ✅ File named `.devcontainer/devcontainer.json`: Works
 ✅ Symlink named `.devcontainer/devcontainer.json`: Works
 ❌ Symlink named `.devcontainer.json`: Doesn't work

**Comment by bHimes** [comment]
Just dropping a +1 on this. Thanks @svenengstrom-vaderstad, I used option 3 as a workaround for now.
